src/modules/dashboard/components/ExpenditurePage/ExpenditureSettings/ExpenditureSettings.tsx

```diff
@@ -76,122 +66,40 @@ const expeditureTypes = [
 
 const displayName = 'dashboard.ExpenditurePage.ExpenditureSettings';
 
-interface Props {
+export interface Props {
   sidebarRef: HTMLElement | null;
   colony: Colony;
 }
 
 const ExpenditureSettings = ({ colony, sidebarRef }: Props) => {
-  const { walletAddress, username } = useLoggedInUser();
-
-  const getDomainColor = useCallback<(domainId: string | undefined) => Color>(
-    (domainId) => {
-      const rootDomainColor: Color = Color.LightPink;
-      const defaultColor: Color = Color.Yellow;
-      if (domainId === String(ROOT_DOMAIN_ID)) {
-        return rootDomainColor;
-      }
-      if (!colony || !domainId) {
-        return defaultColor;
-      }
-      const domain = colony.domains.find(
-        ({ ethDomainId }) => Number(domainId) === ethDomainId,
-      );
-      return domain ? domain.color : defaultColor;
-    },
-    [colony],
-  );
-
-  const renderActiveOption = useCallback<
-    (option: SelectOption | undefined, label: string) => ReactNode
-  >(
-    (option, label) => {
-      const value = option?.value;
-      const color = getDomainColor(value);
-
-      return (
-        <div className={styles.activeItem}>
-          <ColorTag color={color} />
-          <div className={styles.activeItemLabel}>{label}</div>
-        </div>
-      );
-    },
-    [getDomainColor],
-  );
-
-  const filterDomains = useCallback((optionDomain) => {
-    const optionDomainId = parseInt(optionDomain.value, 10);
-    if (optionDomainId === COLONY_TOTAL_BALANCE_DOMAIN_ID) {
-      return false;
-    }
-    return true;
-  }, []);
-
-  const [activeToken, ...tokens] = tokensData;
+  const { values } = useFormikContext<ValuesType>() || {};
+  const expenditureType = values.expenditure;
 
   return (
     <div className={styles.container}>
       <FormSection appearance={{ border: 'bottom' }}>
         <div className={styles.blue}>
+          {/* withDropdownElement - dropdown component created because there is a need to render element outside DOM hierarchy */}
```

Comment:
This comment no longer necessary

Reply:
sure, it is removed (https://github.com/JoinColony/colonyDapp/pull/3833/commits/7bb5d7d9edd766d43626010ec791d5c3bcd43002)